src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryDeployerProperties.java

```diff
@@ -24,10 +24,13 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 /**
+ * Holds configuration properties for connecting to a Cloud Foundry runtime.
+ *
+ * @author Eric Bottard
  * @author Greg Turnquist
  */
 @ConfigurationProperties(prefix = "deployer.cf")
```

Comment:
Hi.  Actually I do generally like to have the PLATFORM last in the name.  This would mean that we should also rename this project to be `spring-cloud-deployer-cloudfoundry`. So ATM doing I'm making this change to be self-consistent with the repository name.  @markfisher originaly like to have the PLATFORM come first but I think is changing his mind.  No one else has offered an opinion.  

So to be consistent across the naming pattern across the board, if we go with `spring-cloud-dataflow-admin-cloudfoundry` (to be known at `spring-cloud-dataflow-cloudfoundry` soon) then this repo should be `spring-cloud-deployer-cloudfoundry` and we can make the property prefix `spring.cloud.deployer.cloudfoundry`.  If we go with `spring-cloud-cloudfoundry-dataflow` then this repo would stay named as it is now.

That is the background.  I presume then you are fine to change this repo name to be `spring-cloud-deployer-cloudfoundry` (and similar change to be made for other deployer impls)